evap/evaluation/models.py

```diff
@@ -813,6 +813,12 @@ def num_voters(self):
             return self._voter_count
         return self.voters.count()
 
+    @property
+    def voter_ratio(self):
+        if self.is_single_result or self.num_participants == 0:
+            return 0
+        return self.num_voters / self.num_participants
```

Comment:
We picked 0 here because we want this to be the ordering, but is this also accurate for `voter_ratio`? I would think that in a single result, the ratio of voters is 100% rather than 0 - or maybe this is just a feeling I have right now

Reply:
Offline discussion: We could probably choose "0" or "1" here. I slightly favor "0" as "1" could indicate "everyone who could have voted voted", and that would be a wrong message e.g. with single results.

Thus, 0 is probably fine for now, but this is basically open for future change.